estate/views/estate_property_actions.xml

```diff
@@ -0,0 +1,96 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="estate_property_action" model="ir.actions.act_window">
+        <field name="name">Properties</field>
+        <field name="res_model">estate.property</field>
+        <field name="search_view_id" ref="estate.estate_property_search_view"/>
+        <field name="view_mode">list,form</field>
+    </record>
+
+    <record id="estate_property_list_view" model="ir.ui.view">
+        <field name="name">estate.list_view</field>
+        <field name="model">estate.property</field>
+        <field name="arch" type="xml">
+            <list string="Properties">
+                <field name="postcode"/>
+                <field name="bedrooms"/>
+                <field name="living_area"/>
+                <field name="expected_price"/>
+                <field name="selling_price"/>
+                <field name="date_availability"/>
+                <field name="property_type_id"/>
+                <field name="property_tag_id" widget="many2many_tags"/>
+            </list>
+        </field>
+    </record>
+
+    <record id="estate_property_form_view" model="ir.ui.view">
+        <field name="name">estate_property_form_view</field>
+        <field name="model">estate.property</field>
+        <field name="arch" type="xml">
+            <form string="New Property">
+                <sheet>
+                    <h1 class="mb32">
+                        <field name="name"/>
+                    </h1>
+                    <group>
+                        <group>
+                            <field name="property_tag_id" widget="many2many_tags"/>
+                            <field name="seller_id"/>
+                            <field name="postcode"/>
+                            <field name="property_type_id"/>
+                            <field name="date_availability"/>
+                            <field name="deadline_date"/>
+                            <field name="validity_days" string="Validity"/>
+                        </group>
+                        <group>
+                            <field name="expected_price"/>
+                            <field name="selling_price"/>
+                        </group>
+                    </group>
+                    <notebook>
+                        <page string="Description">
+                            <group>
+                                <field name="description"/>
+                                <field name="bedrooms"/>
+                                <field name="living_area" string="Living Area(sqm)"/>
```

Comment:
We usually define string in Python so that anytime the field is added to a view, the same string is displayed